src/ngx_rewrite_driver_factory.cc

```diff
@@ -147,10 +147,18 @@ NamedLockManager* NgxRewriteDriverFactory::DefaultLockManager() {
 
 RewriteOptions* NgxRewriteDriverFactory::NewRewriteOptions() {
   NgxRewriteOptions* options = new NgxRewriteOptions(thread_system());
+  // TODO(jefftk): figure out why using SetDefaultRewriteLevel like
+  // mod_pagespeed does isn't enough here -- if you use that instead then
+  // ngx_pagespeed doesn't actually end up defaulting CoreFilters.
+  // See: https://github.com/pagespeed/ngx_pagespeed/issues/1190
```

Comment:
also please add a note that any callers of NewRewriteOptions will get core-filters set.